README.md

```diff
@@ -44,7 +44,5 @@ APP_TOKEN - This is the Trello APP token which you can be obtained by clicking t
 Create a trello TODO board with 2 lists and obtain their IDs and set in the environment variables below
 
 TODO_BOARD_ID 
```

Comment:
We should add some thoughts into this readme detailing that the tests exist and how they can be run, as well as any dependencies such as Firefox/Geckodriver